### ISSUE TITLE
Add mV to `measurement_units` enum list

### ISSUE THREAD

**Issue by NikoSimisiroglou** [IEA-Task-43/digital_wra_data_standard#260]
**Is your feature request related to a problem? Please describe.**
If a slope=1 and offset=0 is applied to a `logger_measurement_config`  this could result in `measurement_units` of `mV` instead of `V` . `mV` is not currently supported in the enum list. 

An example: `measurement_type` pressure will have a value in the timeseries timestep of 2300 and the `measurement_units` is mV not V. 

**Describe the solution you'd like**
Add `mV` to `measurement_units` enum list under `measurement_units_id`

**Additional context**
Something similar was requested among others in #106

**Comment by stephenholleran** [comment]
Hi @NikoSimisiroglou,

Sounds like it is warranted. No harm adding more units to the enum. Are you able to have a go at it by branching off the current dev branch, making the addition and creating a pull request>

Cheers,

**Comment by kersting** [comment]
@stephenholleran, @abohara, @NikoSimisiroglou, and @heikowestermann  I'd also add (m/s)/mV to the list of units in the digital calibration certificate given that if a logger is configured in mV one would need those units for properly handling calibrations on those units. Any opposition?

**Comment by heikowestermann** [comment]
I don't mind, however in practice we at least have never used mV (yet) and default to V instead.

**Comment by kersting** [comment]
@heikowestermann thanks for the quick reply! The problem is that we have seen loggers configured with mV instead of V and we have to make sure that we capture all possible units that could arise as a result of having mV to avoid surprises.

**Comment by heikowestermann** [comment]
@kersting I completely understand and have no objections

**Comment by stephenholleran** [comment]
Hi,

I have created a branch off issue #248 branch as I moved the `measurement_units` into a definition. That branch first needs to be merged into 'dev' before I can create a pull request for this work.

I have added both 'mV' and '(m/s)/mV' to both the wra data model and the DCC.

Are there any other units we want to add while we are at it?

**Comment by NikoSimisiroglou** [comment]
hi @stephenholleran , thank you for creating the branch.  I would include the following units as well deg_C/mV, mbar/mV, hPa/mV, %/mV

**Comment by stephenholleran** [comment]
Hey @NikoSimisiroglou,

Sorry for the slow response.

Those units are not included with the 'V' on it's own yet in either the data model or the digital calibration certificate. So are they really needed yet? I would see that this would explode very quickly as for each measurement type (temperature) we would have to add deg_C/V, deg_C/mV, deg_C/mA, deg_C/Hz similar to what we have for m/s. See screenshot below of the units in the digital calibration certificate.

![image](https://github.com/user-attachments/assets/d54eda47-594e-4a20-b1d5-cf7f91568747)

**Comment by NikoSimisiroglou** [comment]
Hi @stephenholleran, I see your point. For the time being, I would agree to reduce the number of different ratios we include to something reasonable that covers existing cases.  

In our example if  `measurement_type` pressure that has a value in the timeseries timestamp of 2300 with a measurement_units of `mV`, how can we apply a slope to the timestamp value if the units are not available? Here is a print screen from an NRG calibration certificate:

![image](https://github.com/user-attachments/assets/1e7b6578-f542-4544-aaec-89d058ee0118)

We cannot calculate the pressure "P" if we do not have the units of the slope that in this case is hPa / Volt

**Comment by stephenholleran** [comment]
Hi @NikoSimisiroglou,

OK, we can probably just add in the voltage ones for now.

Cheers,

**Comment by stephenholleran** [comment]
@NikoSimisiroglou I have added them in the above commit.

Once the other PR is merged I can create on for this and then it will be a lot easier to review.
